R/class-workbook.R

```diff
@@ -5013,6 +5012,7 @@ wbWorkbook <- R6::R6Class(
     #' Prints the `wbWorkbook` object
     #' @return The `wbWorkbook` object, invisibly; called for its side-effects
     print = function() {
+      # TODO maybe print more information here?
```

Comment:
This is #46 no need for another TODO. Please remove and please feel free to have a look at the issue. I don't really print a lot, but obviously we can always do better. But maybe this is a task for some external package that can use `cli` or whatever.